src/main/java/org/generationcp/middleware/util/Util.java

```diff
@@ -326,6 +326,29 @@ public static Date parseDate(final String date, final String format) throws Pars
 		return formatter.parse(date);
 	}
 
+	/**
+	 * Returns the date object from the specified format
+	 *
+	 * @return date object or null if it cannot be parsed
+	 */
+	public static Date tryParseDate(final String date, final String format) {
+		try {
+			return Util.parseDate(date, format);
+		} catch (final ParseException | NullPointerException e) {
```

Comment:
Several concerns here:
1. We should not be catching NPEs. 
2. Avoid returning null

Reply:
This is mine so I can explain. This is just a convenient method to format the value coming from the db, so we don't have to try/catch every single line where this parsing is needed.

@darla-leafnode @abatac